build.cake

```diff
@@ -298,6 +306,11 @@ Task("BuildAndroidContentProvider").Does(()=>
     {
         appName = "BreakpadSupport";
         BuildAndroidLibrary(appName, "", false);
+        Warning ("Ignoring BreakpadSupport build failure... It's ok.");
```

Comment:
Can you explain that again, it works but it fails, I never understood.

Reply:
No idea, I've searched many forums while implementing that, it just gives an error while the file is being present. My assumption was that it had something to do with relative path 2 levels up from the current folder. Anyway later in the code I make a real check if the `.so` is there so it should be ok.

Reply:
Ok we might want to improve the warning and/or add a comment before to explain.